minio/1.9/README.md

```diff
@@ -0,0 +1,110 @@
+
+Minio server is light enough to be bundled with the application stack, similar to NodeJS, Redis and MySQL. 
+
+This installation uses the single node version of Minio (Minio FS), checkout the [Minio docs](https://docs.minio.io) for more details.
+
+The instructions below use a pinned hostname constraint to ensure the application is always restarted on the same host by Marathon. This allows it to get back to its data but means that you could lose data if that agent goes down. 
+
+- Estimated time for completion: less than 5 minutes. 
+- Target audience:
+    - Operators
+    - Application admins
+    - Developers 
+    - Devops Engineers
+- Scope: Learn to install Minio on DC/OS and learn to use it using Minio command line tool `mc`.
+
+**Table of Contents**:
+
+- [Prerequisites](#prerequisites)
+- [Minio Installation using GUI](#minio-installation-using-gui)
+- [Access Minio Browser](#access-minio-browser)
+- [Install Minio Client](#install-minio-client)
+- [Configure Minio Client](#configure-minio-client)
+- [Uninstalling Minio](#uninstalling-minio)
+- [Explore Further](#explore-further)
+
+## Prerequisites
+
+- DC/OS 1.9 or later
+- [Marathon-LB](https://dcos.io/docs/1.9/usage/service-discovery/marathon-lb/usage/) must be installed and running
+- Identify [IP of the public agent](https://dcos.io/docs/1.9/administration/locate-public-agent/) where Marathon-LB or an available hostname configured to point to the public agent(s) where Marathon-LB is running.
+
+## Minio installation using GUI
+
+- Visit the DC/OS admin page, and click on Universe on the left menu bar. Then click on the Packages tab and search for Minio, click on the ```Install``` button on the right hand side.
+
+![Search & Install Minio](img/search-install.png)
+
+- Then click on ```Advanced Installation```, enter configuration values like the storage and service type you’d like to use with your Minio instance. Finally specify the IP address or hostname of the public agent where Marathon-LB is running under ```networking >> public agent```. Make sure you remove the leading http:// and the trailing / from the IP.
+
+![Public Agent](img/public-agent.png)
+
+- We're ready to install! Click the green ```Review and Install``` button, verify your settings are correct and then click ```Install``` green button.
+
+![Public Agent](img/verify-install.png)
+ 
+- This completes the install process. Once Minio has been deployed, navigate to the IP/hostname you used earlier for Public Agent. You should see the following login page.
+
+## Access Minio Browser
+
+![Minio Browser](img/minio-browser.png)
+
+- We need the access key and secret key from the Minio container logs. Click on ```Services``` Tab, select Minio service and select running instanct(```minio.852ac474-1de8-11e7-9209-ba9616e0f059```). Then go to the ```logs``` tab and copy the accesskey and secretkey.
```

Comment:
"instanct" (not a word :)) should be "service"

And since the parenthetical is just an example, maybe say: (which will look something like ```minio.852ac474-1de8-11e7-9209-ba9616e0f059```). And since we're editing here anyway, the tab is called Logs (with a capital L).